experiments/browser_based_querying/src/hackernews/adapter.ts

```diff
@@ -71,25 +71,17 @@ const _itemPattern = /^https:\/\/news\.ycombinator\.com\/item\?id=(\d+)$/;
 const _userPattern = /^https:\/\/news\.ycombinator\.com\/user\?id=(.+)$/;
 
 function materializeWebsite(fetchPort: MessagePort, url: string): Vertex | null {
-  const itemMatch = url.match(_itemPattern);
-  let ret: any
-  if (itemMatch) {
+  let matcher: RegExpMatchArray | null = null;
+  let ret: any | null = { __typename: 'Website' };
+  if ((matcher = url.match(_itemPattern))) {
```

Comment:
I'm not that familiar with TypeScript, is the double parens `((` required here, or is it just style, or something else?

Reply:
I think it's a prettier thing, coming from the rule: https://eslint.org/docs/latest/rules/no-cond-assign